src/main/java/step3/domain/LottoRanks.java

```diff
@@ -49,14 +37,57 @@ public Long totalPrize() {
         return totalPrize;
     }
 
+    public BigDecimal getCalculatedYield() {
+        return BigDecimal.valueOf(totalPrize())
+            .divide(BigDecimal.valueOf(amount.getAmount()))
+            .setScale(DECIMAL_POINT, RoundingMode.CEILING);
+    }
+
+    public static class Build {
+        Map<LottoRank, CountDown> lottoRanks = new HashMap<>();
+        Amount amount;
+
+        public Build match(LottoNumbersBundle boughtLottoNumberBundle, WinningLotto winningLotto) {
+            for (Map.Entry<LottoRank, CountDown> mapElement : lottoRanks.entrySet()) {
+                LottoRank targetLottoRank = mapElement.getKey();
+                CountDown countDown = mapElement.getValue();
+                int matchCount = boughtLottoNumberBundle.getLottoMatchCountOf(targetLottoRank, winningLotto);
+                countDown.incrementOf(matchCount);
+            }
```

Comment:
`Builder`를 구현해보셨네요 👍 

```suggestion
    public static class Build {
        private Map<LottoRank, CountDown> lottoRanks = new HashMap<>();
        private Amount amount;

        public Build match(LottoNumbersBundle boughtLottoNumberBundle, WinningLotto winningLotto) {
            for (Map.Entry<LottoRank, CountDown> mapElement : lottoRanks.entrySet()) {
                LottoRank targetLottoRank = mapElement.getKey();
                CountDown countDown = mapElement.getValue();
                int matchCount = boughtLottoNumberBundle.getLottoMatchCountOf(targetLottoRank, winningLotto);
                countDown.incrementOf(matchCount);
            }
```